### DISCUSSION TITLE
問問你各位的期末專題

### DISCUSSION THREAD

**Discussion by zoo100130** [1101-GS4719-C/Python-Community-GS4719-C#2283] [General]
1.啊啊啊啊我想不到要做什麼
2.欸嘿我是遷移學習大師: 直接把別堂課的拿來做
3.心向遠方 胸懷大海: 這玩意兒我都準備拿去推甄了哼哼
4.我是大玩家 想到啥再做啥 just for fun 
5.555我想要小夥伴陪我一起作QAQ

歡迎各位分享介紹大家想做的專題方向，說不定會有類似題目的同學可以互相交流喔 
兌卦:君子以文會友，以友輔仁
GS4719AL:同學以python增能，以能踐業

**Comment by christaaah** [comment]
我的話一開始也不知道要做什麼，對於專題要做到的難度也拿不太準，但是透過和gpt聊天後，發現了一項可以跟我最近的興趣做結合的題目，但是難度不太確定，但是朋友說只要能學到新東西而且自己感興趣那就可以做，所以我就做了。現在是感覺沒有我想像中那麼難，果然什麼事情都是先做再說，反正過不了就鬼轉（不是。

**Comment by kzx-creat** [comment]
我是製作網站，網站有加特定功能的人工智慧，主題有跟別堂課重疊，但專題製作的重點兩堂課稍有不同。網站還在做，我覺得小困難，需要花點時間看懂。幸好有GPT，有啥問題就問它 :)
